Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -44,7 +45,7 @@ namespace Emby.Server.Implementations.Session
     /// <summary>
     /// Class SessionManager.
     /// </summary>
-    public class SessionManager : ISessionManager, IDisposable
+    public sealed class SessionManager : ISessionManager, IAsyncDisposable
```

Comment:
You should implement IDisposable when implementing IAsyncDisposable

Reply:
Since the lifetime of this object is managed by the service collection, this is guaranteed to be correctly disposed by the container. 

See https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-disposeasync